src/components/date-range-picker.ts

```diff
@@ -63,6 +63,34 @@ const Component = Vue.extend({
       type: Boolean,
       default: false,
     },
+    weekdaySu: {
+      type: String,
+      default: "Su",
+    },
+    weekdayMo: {
+      type: String,
+      default: "Mo",
+    },
```

Comment:
Is this a dumb way to solve this? 

I don't think I can pass an array through to this component, so being limited to strings it seems I can either have one attribute per day (and one per month), or I can use JSON to serialize and deserialize it on every render. 

Not sure what would be preferred. Or maybe I'm missing some alternative.